From c91ca44910a7383e38b14dad2f615530043f373c Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@nxp.com>
Date: Wed, 12 Oct 2016 16:08:30 +0800
Subject: [PATCH 1204/5242] MLK-13312-1 ARM: imx6sll: refine USB support

commit  c46bb7ea9aeada12eee714f999b15b118ed03727 from
https://source.codeaurora.org/external/imx/linux-imx.git

The imx6sll is much like imx6ul, so add imx6ul compatible string for it.

Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sll.dtsi |   15 ++++++++++-----
 1 file changed, 10 insertions(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sll.dtsi b/arch/arm/boot/dts/imx6sll.dtsi
index 4a4a07f..8da2b38 100644
--- a/arch/arm/boot/dts/imx6sll.dtsi
+++ b/arch/arm/boot/dts/imx6sll.dtsi
@@ -500,7 +500,8 @@
 			};
 
 			usbphy1: usbphy@020c9000 {
-				compatible = "fsl,imx6sll-usbphy", "fsl,imx23-usbphy";
+				compatible = "fsl,imx6sll-usbphy", "fsl,imx6ul-usbphy",
+						"fsl,imx23-usbphy";
 				reg = <0x020c9000 0x1000>;
 				interrupts = <GIC_SPI 40 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&clks IMX6SLL_CLK_USBPHY1>;
@@ -509,7 +510,8 @@
 			};
 
 			usbphy2: usbphy@020ca000 {
-				compatible = "fsl,imx6sll-usbphy", "fsl,imx23-usbphy";
+				compatible = "fsl,imx6sll-usbphy", "fsl,imx6ul-usbphy",
+						"fsl,imx23-usbphy";
 				reg = <0x020ca000 0x1000>;
 				interrupts = <GIC_SPI 41 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&clks IMX6SLL_CLK_USBPHY2>;
@@ -655,7 +657,8 @@
 			ranges;
 
 			usbotg1: usb@02184000 {
-				compatible = "fsl,imx6sll-usb", "fsl,imx27-usb";
+				compatible = "fsl,imx6sll-usb", "fsl,imx6ul-usb",
+						"fsl,imx27-usb";
 				reg = <0x02184000 0x200>;
 				interrupts = <GIC_SPI 43 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&clks IMX6SLL_CLK_USBOH3>;
@@ -669,7 +672,8 @@
 			};
 
 			usbotg2: usb@02184200 {
-				compatible = "fsl,imx6sll-usb", "fsl,imx27-usb";
+				compatible = "fsl,imx6sll-usb", "fsl,imx6ul-usb",
+						"fsl,imx27-usb";
 				reg = <0x02184200 0x200>;
 				interrupts = <GIC_SPI 42 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&clks IMX6SLL_CLK_USBOH3>;
@@ -683,7 +687,8 @@
 
 			usbmisc: usbmisc@02184800 {
 				#index-cells = <1>;
-				compatible = "fsl,imx6sll-usbmisc", "fsl,imx6q-usbmisc";
+				compatible = "fsl,imx6sll-usbmisc", "fsl,imx6ul-usbmisc",
+						"fsl,imx6q-usbmisc";
 				reg = <0x02184800 0x200>;
 			};
 
-- 
1.7.9.5

